cmd/porter/bundle.go

```diff
@@ -31,14 +31,23 @@ func buildBundleCommands(p *porter.Porter) *cobra.Command {
 }
 
 func buildBundleCreateCommand(p *porter.Porter) *cobra.Command {
-	return &cobra.Command{
-		Use:   "create",
+	cmd := &cobra.Command{
+		Use:   "create [bundle-name]",
 		Short: "Create a bundle",
-		Long:  "Create a bundle. This generates a porter bundle in the current directory.",
+		Long:  "Create a bundle. This generates a porter bundle with the directory with the specified name.",
+		Args:  cobra.MaximumNArgs(1), // Expect at most one argument for the bundle name
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return p.Create()
+			bundleName := "mybundle" // Default value
```

Comment:
Is this necessary? This command will error if bundle name is not passed which I believe is what we want. 

```bash
porter create // will error out
porter create mybundle //will not fail 
```

Reply:
```suggestion
			var bundleName string 
```

Reply:
@troy0820 currently it will not throw any error. It instead will create a bundle with the default name "mybundle". But if we want to throw an error if the argument isn't passed, then sure, I will fix it.

Reply:
@schristoff @phillipgibson Do we want the behavior to error _(error being a word used to describe not executing the command but show the usage)_ when the argument for the create bundle isn't given to make the user do this? 

```cli
porter create [bundle]
```

Or do we want to create the bundle with the default `mybundle` in the directory if the argument isn't given?

Reply:
>  But if we want to throw an error if the argument isn't passed, then sure, I will fix it.

@sbdtu5498 
I'm not quite sure but I want to see what is the desired behavior that we want and asking the community.  From the issue I believe we want to emulate the same behavior as the below clis. 


`helm create` requires 1 argument 
`cargo new ` requires 1 argument

Reply:
@troy0820 sure thing! Once we get more reviews on how it should be handled, I will add the required changes.

Reply:
My bias is towards helm, and feeling that most users of our product are familiar with helm. 
In that case, let's force the user to do the hard thing, and pick a name. Error if they don't pass in an argument, because making assumptions can cause issues down the line. :)

Reply:
Sure thing, this seems feasible.